src/helperFunctions/getCloudinaryImageUrl.ts

```diff
@@ -0,0 +1,18 @@
+const cloudinaryImageUrl = `https://res.cloudinary.com/realdevsquad/image/upload`;
+const userBaseUrl = `https://api.realdevsquad.com`;
+
+export const getCloudinaryImgURL = async (userName: string,config:string) => {
+    const memberDataUrl = `${userBaseUrl}/users/${userName}`
```

Comment:
you can either use uppercase convention here or make `Url` uppercase like this- `URL`. Not a much of a naming convention but looks better though.

Reply:
Yes `URL` seems better. Modified it now. Thanks.